### ISSUE TITLE
online guide is not in accordance with .cxx and .externalNativeBuild no longer exists.

### ISSUE THREAD

**Issue by graph** [android-ndk/ndk#1070]
source: https://developer.android.com/ndk/guides/cmake

claims that c++ builds go here:
```
<project-root>/<module-root>/.externalNativeBuild/cmake/<build-type>/<ABI>/
```

but instead it goes into a `.cxx` directory. If you are changing the directory it should be noted more verbosely in the documentation as various scripts already depend on the existing .externalNativeBuild directory. And if you are changing the default please move the default to the build directory instead of the project directory thank you.

**Comment by DanAlbert** [comment]
I've just sent a patch to update that doc. Ideally it wouldn't be a part of that doc at all since it's really the gradle plugin that's doing that so it should be a part of the gradle docs, but that's a much larger undertaking so I've just fixed this for now.

> And if you are changing the default please move the default to the build directory instead of the project directory thank you.

That's the gradle plugin, not the NDK, so we can't do anything about it. File a Studio bug following https://b.android.com for gradle plugin issues.

**Comment by alexcohn** [comment]
@DanAlbert if you are fixing these doc pages, maybe you could also mention there the underdocumented misfeature of CMake integration, that the prebuilt shared libraries are not copied to APK automatically (as with ndk-build), but must be placed according to jniLibs settings of build.gradle.

**Comment by DanAlbert** [comment]
@alexcohn Landed back here for totally unrelated reasons, but that is fixed as of 4.0 :)
